pkg/v20/resource/appmigration/create.go

```diff
@@ -62,3 +70,52 @@ func (r *Resource) newChartSpecsToMigrate() []key.ChartSpec {
 
 	return chartSpecsToMigrate
 }
+
+func addCordonAnnotations() map[string]string {
+	return map[string]string{
+		annotation.CordonReason:    "cordoning chartconfig CR for migration to app CR",
+		annotation.CordonUntilDate: time.Now().Add(1 * time.Hour).Format("2006-01-02T15:04:05"),
```

Comment:
What is our thinking of one hour here? I think having that in some key function makes sense because searching for this one hour value here could be a pain in the future.

Reply:
The hour is to give time for the migration to happen. If it isn't complete after an hour the cordon alert will fire and the reason can be investigated.

Yes sure I'll move it to a key function.